regulatory_compliance/appconfiguration.sp

```diff
@@ -40,4 +70,64 @@ query "app_configuration_private_link_used" {
       azure_app_configuration as a,
       azure_subscription as sub;
   EOQ
+}
+
+query "app_configuration_restrict_public_access" {
+  sql = <<-EOQ
+    select
+      a.id as resource,
+      case
+        when public_network_access = 'Enabled' then 'alarm'
```

Comment:
@khushboo9024 I guess we have to add a check for private-end-point along with the `public_network_access` attribute?

![image](https://github.com/turbot/steampipe-mod-azure-compliance/assets/26873346/63b2f247-eca2-4a8a-be63-6e8d323e7688)